src/os/unix/unix.cpp

```diff
@@ -49,7 +49,7 @@
 #endif
 
 #if defined(__APPLE__)
-#	if defined(WITH_SDL)
+#	if defined(WITH_SDL) || defined(WITH_SDL2)
```

Comment:
i imagine this is an SDL1-only thing, as it did funky things with main(), but SDL2 no longer does this